src/components/accountView/accountView.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, EmptyState, Grid, Row } from 'patternfly-react';
+import { reduxTypes, store } from '../../redux';
```

Comment:
... from what i can tell it connecting the component doesn't seem to be needed to dispatch. going a little bit off progressively enhancing the components, when it comes around it may need to be added back in